### DISCUSSION TITLE
HyperHDR v20b1 Black screen, LED's shine a bit.

### DISCUSSION THREAD

**Discussion by wanted8x** [awawa-dev/HyperHDR#696] [General]
Hi,
thanks for your Work.
I updated my HyperHDR from v19.
All works fine but in v20b1 black areas LED's shine a bit. In v19 Black areas LED's are off. 

Its a feature or a bug? I am now back in v19.

Thank you

**Comment by awawa-dev** [comment]
Hi
Did you try to set minimal  backlight to 0 in v20 processing tab?

**Reply by wanted8x** [reply]
Thats it thank you so much !! :))
